Configuration/AlCa/python/autoCond.py

```diff
@@ -76,4 +74,4 @@
 autoCond['upgradePLS150ns']  = ( autoCond['run2_mc_50ns'] )
 autoCond['upgrade2017']      = ( autoCond['phase1_2017_design'] )
 autoCond['upgrade2019']      = ( autoCond['phase1_2019_design'] )
-autoCond['upgradePLS3']      = ( autoCond['phase2_design'] )
+autoCond['upgradePLS3']      = ( autoCond['phase2_realistic'] )
```

Comment:
thanks @lgray I was wondering if there is a way of letting all these people here: https://github.com/cms-sw/cmssw/search?p=1&q=upgradePLS3&type=Code&utf8=%E2%9C%93 know that the content of their conditions is suddenly not the one they had used to have.